pallets/asset/src/lib.rs

```diff
@@ -210,7 +210,7 @@ pub struct SecurityToken<U> {
     pub owner_did: IdentityId,
     pub divisible: bool,
     pub asset_type: AssetType,
-    pub primary_issuance_agent: Option<IdentityId>,
+    pub primary_issuance_agent: IdentityId,
```

Comment:
This changes the data structure and requires schema change, storage migration, and relevant updates in the UIs. I am not convinced that it's worth it at this point given that we will be reworking agents in the next iteration anyway.

Reply:
Reverted back to an Option